src/add-and-remove-task.test.js

```diff
@@ -0,0 +1,14 @@
+import localStorageController from './__mocks__/localStorage-controller';
+import taskController from './add-and-remove-task';
+
+jest.mock('./localStorage-controller');
+
+describe('to do list add and delete', () => {
+  test('Add new task', () => {
+    const tasksArr = localStorageController.getDataFromLocalStorage('dum');
+    expect(taskController.addTask(`task${tasksArr.length + 1}`)).toHaveLength(3);
+  });
+  test('Delete task', () => {
+    expect(taskController.deleteTask(0)).toHaveLength(2);
+  });
```

Comment:
- [ ] As the requirements say you should test all the pure functions, so you need to test also the edit, complete: true/false, and also clear all completed.

Reply:
This issue is well resolved! ✔️